README.md

```diff
@@ -1,3 +1,11 @@
-Take this course if you'd like to learn how to write and publish your own course on GitHub Learning Lab.
+Learning Lab helps people build real world skills through hands-on activities. This course streamlines the course building process, and gives you all the tools you need to help others learn. 
 
-You'll need to be a owner of this organization in order to publish the course you create.
\ No newline at end of file
+Register for this course to get the starter files needed to build your Learning Lab course, or complete the course steps for a fully guided experience. If you choose to complete the course, you'll be able to:
```

Comment:
Minor nit, but do you think it would be organized better to move "Register for this course to get the starter files needed to build your Learning Lab course, or complete the course steps for a fully guided experience." into the first paragraph?

Reply:
nvm